docs/learn/learn-elastic-scaling.md

```diff
@@ -0,0 +1,77 @@
+---
+id: learn-elastic-scaling
+title: Polkadot's Elastic Scaling
+sidebar_label: Elastic Scaling
+description: Pushing Parachains' Boundaries Below 6 seconds Execution Time.
+keywords: [elastic scaling, parachains, coretime, blockspace]
+slug: ../learn-elastic-scaling
+---
+
+The path of parablocks from their creation to their inclusion into the relay chain (discussed in the
+[Parachain Protocol Page](./learn-parachains-protocol.md)) span two domains: the parachain's and
+relay chain's roles. Scaling the Polkadot protocol involves consideration of how parablocks are
+validated, processed, secured, made available for additional checks, and finally included in the
+relay chain as part of its final state
+
+[Asynchronous backing](./learn-async-backing.md) is the first implementation of the parachain
+protocol that alters how parachains produce blocks and how the relay chain processes their data.
+This upgrade will allow parachains to take up to 2 seconds execution time (for producing
+parablocks), while the relay chain will be able to include a parablock every 6 seconds.
+
+With asynchronous backing enabled, parachains can theoretically produce parablocks in the sub-second
+time range. However, to accomplish this, asynchronous backing must temporarily **hold** blocks in
+[unincluded segments](./learn-async-backing.md#unincluded-segments), while parablocks can be
+included into the relay chain every 6 seconds per core. This is the bottleneck that elastic scaling
+addresses. With elastic scaling, parachains can use multiple cores to include multiple parablocks
+within the same relay chain block.
+
+Take, for example, a parachain that wants to submit four parablocks to the relay chain. Without
+elastic scaling, it will take 24 seconds to include all of them through one core. Remember that a
```

Comment:
> it will take 24 seconds

24 seconds or 30?

Reply:
the chain starts at R0, at R1 is 6s, and R4 is 30 s